html/index.html

```diff
@@ -142,12 +142,13 @@ <h3 class="title is-size-4-desktop is-size-5">Enter address and amount (0.001-0.
                                         <img id="captcha_img" src="/captcha?<%- new Date().getTime() %>"/>
                                     </div>
                                     <div class="control">
-                                        <input class="input" id="captcha" name="captcha" type="input" placeholder="captcha">
+                                        <input class="input" id="captcha" name="captcha" type="input" placeholder="captcha" tabindex="3">
```

Comment:
Shouldn't this be tabindex 4?

Reply:
OK, will do the SEND button 3 and those optional ones 4. Good idea.

Reply:
See e8d4f13 (force-pushed, the previous contained errors)